x-pack/plugins/cases/public/containers/use_get_case_user_actions.tsx

```diff
@@ -247,14 +55,12 @@ export const useGetCaseUserActions = (caseId: string, caseConnectorId: string) =
         : [];
```

Comment:
I can't add the comment in the line I want because there were no changes but check line 46.

`export const useGetCaseUserActions = (caseId: string, caseConnectorId: string) => {`

this hook gets the `caseConnectorId` only for the useQuery key in line 50. That is a bit unnecessary no?

Reply:
You are right. I am not sure why the connector ID was bound to the caching key. It makes more sense to be bound only to the case ID. When we update we invalidate the cache so it should be fine.